satellytes/satellytes.com
# feat: added rss-feed for blog article

pr contains the RSS fead for the blog posts

to test: 
- Blog post with "Blog abonnieren"-Link at the bottom of the page: [https://satellytescommain-featrssfeed.gtsb.io/blog/how-consistency-helps-you-to-optimize-gatsby-urls/](https://satellytescommain-featrssfeed.gtsb.io/blog/how-consistency-helps-you-to-optimize-gatsby-urls/)
- the xml-file itself: [https://satellytescommain-featrssfeed.gtsb.io/blog/rss.xml](https://satellytescommain-featrssfeed.gtsb.io/blog/rss.xml)

technical notes:
- implemented based on [https://www.gatsbyjs.com/docs/how-to/adding-common-features/adding-an-rss-feed/#customizing-the-rss-feed-plugin](https://www.gatsbyjs.com/docs/how-to/adding-common-features/adding-an-rss-feed/#customizing-the-rss-feed-plugin)
- RSS-Link page of SZ: [https://www.sueddeutsche.de/service/updates-mit-rss-uebersicht-aller-rss-feeds-fuer-sz-de-sz-magazin-und-jetzt-de-1.393950](https://www.sueddeutsche.de/service/updates-mit-rss-uebersicht-aller-rss-feeds-fuer-sz-de-sz-magazin-und-jetzt-de-1.393950) (just as an example)

(in general) how to follow blog feed:
A RSS-Feed contains a summary of updates from the website. With an RSS-Feed Reader you can subscribe to numerous feeds and it provides a simple overview of the pages you follow. (So you just have to look at the Feed-Reader instead of visiting each websites)

- you can search for the page in your feed-reader. Looks like that:
<img width="700" alt="Screenshot 2021-08-19 at 08 38 56" src="https://user-images.githubusercontent.com/72851928/130028164-d7d4a7eb-ddd6-43f8-8f9d-2713bc6c0d8a.png">
And the posts are shown below
<img width="500" alt="Screenshot 2021-08-19 at 08 39 13" src="https://user-images.githubusercontent.com/72851928/130028138-b43fb630-1925-421a-8cad-33d533b73013.png">


- or you can click on the "blog abonnieren"-link below a post which leads to the rss.xml. If a browser extension for RSS-Reading is installed, a pop-up should appear:
<img width="429" alt="Bildschirmfoto 2021-08-19 um 09 45 47" src="https://user-images.githubusercontent.com/72851928/130029073-e5d10534-62dc-4b7c-8bf8-c59207898202.png">
